sky/provision/aws/instance.py

```diff
@@ -101,6 +105,40 @@ def _cluster_name_filter(cluster_name_on_cloud: str) -> List[Dict[str, Any]]:
     }]
 
 
+def _ec2_call_with_retry_on_server_error(ec2_fail_fast_fn: Callable[..., _T],
+                                         log_level=logging.DEBUG,
+                                         **kwargs) -> _T:
+    # Here we have to handle 'RequestLimitExceeded' error, so the provision
+    # would not fail due to request limit
+    # issues.
+    # Here the backoff config (5, 12) is picked at random and does not
+    # have any special meaning.
+    backoff = common_utils.Backoff(5, 12)
```

Comment:
```suggestion
    backoff = common_utils.Backoff(initial_backoff=5, max_backoff_factor=12)
```
nit: use kwargs for literal